functions.php

```diff
@@ -0,0 +1,212 @@
+<?php
+
+
+function checkSingleCardAndGetAllInfo(string $cardTitle): array {
+    $connectionString = 'mysql:host=db; dbname=mtg_cards';
+    $dbUsername = 'root';
+    $dbPassword = 'password';
+    $db = new PDO ($connectionString, $dbUsername, $dbPassword);
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $queryString = "SELECT * FROM `cards` WHERE `title` LIKE '$cardTitle'";
+    $query = $db->prepare($queryString);
+    $query->execute();
+    return $result = $query->fetch();
```

Comment:
should be just `return $query->fetch();`

Reply:
This function has been deleted.